pom.xml

```
@@ -9,6 +9,7 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
+        <jdk.version>23</jdk.version>
```

It is better to use LTS version
```suggestion
        <jdk.version>21</jdk.version>
```